### DISCUSSION TITLE
Raytracer

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/small-raytracer#4] [Q&A] [answered]
What are the instructions for the end users?

**Comment by Stagyrite** [comment, marked as answer]
### Install
To install this piece of software you have to get a C compiler. This is available on Cygwin, if you run Windows.
```
git init
git remote add origin git@github.com:Stagyrite/small-raytracer.git
git pull origin master
make
```
It'll compile the executable file that you can run with a ./raytracer command. It writes its output to a 'test.bmp' file.
### Test result
![test](https://github.com/eltikia/small-raytracer/assets/55529808/4adcec57-4b7d-42fe-83e0-1ea0f8ae6d9b)
